cluster/ci/config/roles.yaml

```diff
@@ -868,3 +868,52 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: system:image-puller
+
+- kind: Namespace
+  apiVersion: v1
+  metadata:
+    name: open-cluster-management
+    annotations:
+      purpose: "RedHat Advanced Cluster Management Build Artifacts"
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: ocm-admins
+  users:
+  - ray-harris
+  - arewm
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: admin
+    namespace: open-cluster-management
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: ocm-admins
+  - kind: Group
+    name: ci-admins
```

Comment:
No need to add ci-admins ... they are admins already.

Reply:
I'll take that one out.